values.yaml

```diff
@@ -0,0 +1,6 @@
+appName: loki-alert-rule-group-controller
```

Comment:
I've usually used Release name. Is this a better approach?

Reply:
ya, I can do that - this value drives the endpoint for metacontroller, so it would also move every time we upgrade

Reply:
I think as long as we deploy it with the same name in ArgoCD it would persist, right?

Reply:
I'll give it a shot and let you know